examples/turbo-rspack-mf/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ze-starter-turbo",
+  "private": true,
+  "scripts": {
+    "build": "turbo build",
+    "check": "biome check --write",
+    "dev": "turbo dev",
+    "format": "biome format --write",
+    "lint": "turbo lint"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "^1.9.3",
+    "turbo": "^2.3.1",
+    "typescript": "5.5.4"
+  },
+  "packageManager": "pnpm@8.15.6",
+  "engines": {
+    "node": ">=18"
+  }
+}
```

Comment:
Shane, is it possible to add all application dependencies on the root `package.json` for the project?

This will facilitate version bumping in the future.

Reply:
We can, I'm not a fan of this though. Can read more about it [here](https://turbo.build/repo/docs/crafting-your-repository/managing-dependencies#best-practices-for-dependency-installation)

Reply:
I agree its not best practices but this is just to showcase Zephyr and whatever tech we want.

It's easier for us to update them all if they are just on root.